testSuite/scripts/test_upload_block_blob.py

```diff
@@ -409,7 +409,7 @@ def test_upload_block_blob_include_flag(self):
         # uploading the directory with sub-dir in the include flag.
         result = util.Command("copy").add_arguments(dir_n_files_path).add_arguments(util.test_container_url). \
             add_flags("recursive", "true").add_flags("log-level", "info") \
-            .add_flags("include-path", "sub_dir_include_flag_set_upload/*").add_flags("output-type",
+            .add_flags("include-path", "sub_dir_include_flag_set_upload/").add_flags("output-type",
```

Comment:
Out of curiosity, why was it necesary to change the tests here?  I didn't _notice_ anything that would have changed behaviour in this regard, in the code.

Reply:
include-path does not use wildcards when present. Thus, this test would fail.